docs/docs/tutorial/part-2/index.mdx

```diff
@@ -750,6 +750,14 @@ const Layout = ({ pageTitle, children }) => {
 export default Layout
 ```
 
+<Announcement>
+
+**Syntax Hint:** In CSS, the convention is to name classes using kebab case (like `.nav-links`). But in JavaScript, the convention is to name variables using camel case (like `navLinks`).
```

Comment:
You use `kebab case` & `camel case` + `kebab-case` & `camel-case` here

Reply:
Ah yeah, that's a grammar weirdness. When you use a two-word phrase as an adjective before a noun, you hyphenate it. But if it's just the two-word phrase on its own, you don't.

https://www.grammarbook.com/punctuation/hyphens.asp

Reply:
TIL. Thanks for the info!